internal/generator/vector/input/source.go

```diff
@@ -42,20 +38,20 @@ var (
 			Build()
 )
 
-// NewViaQ creates an input adapter to generate config for ViaQ sources to collect logs excluding the
+// NewSource creates an input adapter to generate config for ViaQ sources to collect logs excluding the
 // collector container logs from the namespace where the collector is deployed
-func NewViaQ(input logging.InputSpec, collectorNS string, resNames *factory.ForwarderResourceNames, op framework.Options) ([]framework.Element, []string) {
+func NewSource(input logging.InputSpec, collectorNS string, resNames *factory.ForwarderResourceNames, op framework.Options) ([]framework.Element, []string) {
 	els := []framework.Element{}
 	ids := []string{}
 	switch {
 	case input.Name == logging.InputNameApplication:
-		els, ids = NewViaqContainerSource(input, collectorNS, "", infraExcludes)
+		els, ids = NewContainerSource(input, collectorNS, "", infraExcludes, logging.InputNameApplication, logging.InfrastructureSourceContainer)
```

Comment:
Maybe make sense add new or rename const because it can confuse Input is `application` but source type is `InfrastructureSourceContainer`

Reply:
Agree. I'll address in a separate PR